asm/Kptfile

```diff
@@ -1,123 +1,221 @@
 apiVersion: kpt.dev/v1alpha1
 kind: Kptfile
 metadata:
-  name: asm
+  name: asm-istiod-15
+packageMetadata:
+  shortDescription: ASM Networking 1.5
 openAPI:
   definitions:
-    io.k8s.cli.setters.gcloud.compute.location:
-      type: string
+    io.k8s.cli.setters.meshca-address:
       x-k8s-cli:
         setter:
-          name: gcloud.compute.location
-          value: us-central1-c
-    io.k8s.cli.setters.gcloud.compute.network:
-      type: string
+          name: meshca-address
+          value: meshca.googleapis.com:443
+    io.k8s.cli.substitutions.meshca-address:
       x-k8s-cli:
-        setter:
-          name: gcloud.compute.network
-          value: default
-    io.k8s.cli.setters.gcloud.compute.subnetwork:
-      type: string
+        substitution:
+          name: meshca-address
+          pattern: MESHCA
+          values:
+          - marker: MESHCA
+            ref: '#/definitions/io.k8s.cli.setters.meshca-address'
+    io.k8s.cli.setters.trafficdirector-address:
       x-k8s-cli:
         setter:
-          name: gcloud.compute.subnetwork
-          value: default
-    io.k8s.cli.setters.gcloud.container.cluster.releaseChannel:
-      type: string
+          name: trafficdirector-address
+          value: trafficdirector.googleapis.com:443
+    io.k8s.cli.substitutions.trafficdirector-address:
```

Comment:
consider removing this substitution. It is a duplicate with io.k8s.cli.setters.trafficdirector-address.